module-4/backend/week_four.md

```diff
@@ -5,15 +5,28 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's your favorite project management tool?
+-Waffle.io by far. Has everything I need and is not overly complicated. 
 2. Why do we create staging environments?
+Its important to create staging enviroments so we can keep our production and development serperate and clean. Things on the back end may be different depending on enviroment.
```

Comment:
it's important to replicate our production environment as closely as possible and test that our code is functioning properly on staging before actually pushing new changes to production (where our users are actively using our app)